extensions/flyway/runtime/src/main/java/io/quarkus/flyway/runtime/QuarkusResourceProvider.java

```diff
@@ -0,0 +1,56 @@
+package io.quarkus.flyway.runtime;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+import org.flywaydb.core.api.ResourceProvider;
+import org.flywaydb.core.internal.resource.LoadableResource;
+import org.flywaydb.core.internal.util.StringUtils;
+import org.jboss.logging.Logger;
+
+/**
+ * This implementations of this class are very similar to those of {@link org.flywaydb.core.internal.scanner.Scanner}
```

Comment:
```suggestion
 * This implementations of this class are very similar to those of {@link org.flywaydb.core.internal.scanner.Scanner}
```
There is something weird here with `This` being singular and the rest being plural.

Reply:
Indeed, it should be better now